activejob/test/cases/logging_test.rb

```diff
@@ -368,4 +368,109 @@ def test_enqueue_all_job_logging
     ActiveJob.perform_all_later(LoggingJob.new("Dummy"), HelloJob.new("Jamie"), HelloJob.new("John"))
     assert_match(/Enqueued 3 jobs to .+ \(2 HelloJob, 1 LoggingJob\)/, @logger.messages)
   end
+
+  def test_enqueue_log_level
+    @logger.level = WARN
+    HelloJob.perform_later "Dummy"
+    assert_no_match(/Enqueued HelloJob \(Job ID: .*?\) to .*?:.*Dummy/, @logger.messages)
```

Comment:
```suggestion
    assert_no_match(/HelloJob/, @logger.messages)
```

When using negative matcher, it's preferable to use the widest possible matcher.

Otherwise it's too easy to break without realizing because the format changed a bit.

Reply:
Alternatively, it can make sene if you explicitly use the same regexp and trigger code in both `assert_match` and `assert_no_match` with only the log level changing.

Reply:
Ok, I just went with the less greedy regexp for the negative matches.